src/schema/v1/fields/money.ts

```diff
@@ -105,6 +109,37 @@ const money = ({ name, resolve }) => ({
   }),
 })
 
+export const Price = new GraphQLObjectType<any, ResolverContext>({
+  name: "Price",
+  fields: {
+    minor: {
+      type: GraphQLFloat,
```

Comment:
i wonder if we should make the `minor` unit `Int` and not `Float`

Reply:
I made that change.